docs/SlavaUkraini/examples.md

```diff
@@ -33,6 +33,14 @@ description="In this pretty tutorial, you'll learn how to easily define your own
 %}
 
 
+{% include tutorial_link.html
+page="accessibility/"
+thumbnail="accessibility/thumbnail.png"
```

Comment:
This may not be the most exciting [image](https://github.com/Leaflet/Leaflet/pull/8081/files#diff-d31b250c75272a3ce989954d74f6ebdbb9d0a970757e06c44551f7959b4aed73), but it's something.